src/libraries/System.Runtime.Serialization.Xml/ref/System.Runtime.Serialization.Xml.cs

```diff
@@ -456,6 +540,7 @@ public abstract partial class XmlDictionaryWriter : System.Xml.XmlWriter
     {
         protected XmlDictionaryWriter() { }
         public virtual bool CanCanonicalize { get { throw null; } }
+        public override void Close() { }
```

Comment:
This is another subtle bug in the reference assembly. If the user code implements this type and calls base implementation in Close method override, wrong implementation of the base Close method would be called before this change.